CRM/Upgrade/Incremental/sql/5.32.alpha1.mysql.tpl

```diff
@@ -1 +1,8 @@
 {* file to handle db changes in 5.32.alpha1 during upgrade *}
+
+UPDATE civicrm_job SET name = 'Enable disable relationships',
+description = ' Enables relationships where start date <  = today (ie those relationships whose start date is today / in the past).
+Disables relationships that have expired (ie. those relationships whose end date is in the past).',
+api_action = 'enable_disable_relationships'
+WHERE api_action = 'disable_expired_relationships';
+
```

Comment:
If you change this wouldn't it break for anyone who already calls it in a cron script? Or am I thinking of something else? This is the `-a` param in bin/cli.php right?

Reply:
Yep It would break but it had to be renamed as its functionality is getting changed to a major extent and I am not sure how these types of changes are announced in civicrm

Reply:
I see your point just it's sometimes difficult to communicate that to all affected parties and it's a longstanding fixture. Maybe a good analogy is api Contact.create. The word `create` is inaccurate and what it really does is `create-or-update`. But via documentation and examples nobody really gets confused. So I'd still vote for leaving anything used programmatically like the api_action unchanged.

Reply:
I have changed it as per your suggestion leaving the api_action unchanged but changed the description and name to explain the working of the scheduled job